node/src/main/scala/coop/rchain/node/web/WebApiRoutes.scala

```diff
@@ -70,9 +74,12 @@ object WebApiRoutes {
         handleRequest[Unit, B](req, _ => f)
     }
 
-    // TODO: Create generic encoders/decoders for
-    // ADT's with discriminator field
```

Comment:
Please leave these comments. JSON with discriminator field for case classes is I think better for clients to work with.